submissions/Jigsaff/a-tiny-JS-world-post-OOP/index.js

```diff
@@ -0,0 +1,112 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: _put repo URL here_
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+
+class Organism {
+    constructor(type, legs, hands, name, gender, saying) {
+        this.type = type;
+        this.legs = legs;
+        this.hands = hands;
+        this.name = name;
+        this.gender = gender;
+        this.saying = saying;
+        this.friends = [];
+    }
+
+    addFriends(...Friends) {
+        Friends.forEach(({name}) => this.friends.push(name));
+        return this.friends;
+    }
+
+    toString() {
+        let strings = [
+            `type: ${this.type}; `,
+            `legs: ${this.legs}; `,
+            `hands: ${this.hands}; `,
+            `name: ${this.name}; `,
+            `gender: ${this.gender}; `,
+            `saying: ${this.saying}; `,
+        ];
+
+        if (this.friends.length > 0) {
+            strings.push(`friends: ${this.friends}; `);
+        }
```

Comment:
You need keep the order. [Task](https://github.com/OleksiyRudenko/a-tiny-JS-world#the-job)

> List inhabitants using project built-in print(message) function. Each list entry should look like human; John; male; 2; 2; Hello world!
if inhabitant has no hands then skip it or report 0 or undefined, it is totally up to you